CMakeLists.txt

```diff
@@ -20,7 +19,17 @@ ADD_DEFINITIONS('-DBUILD_KERNEL_RELEASE="${BUILD_KERNEL_RELEASE}"')
 
 ENABLE_TESTING()
 
+IF(NOT DEFINED WITH_DOCS)
+    SET(WITH_DOCS "YES")
```

Comment:
IMO fajnie by było, gdyby to się defaultowało do `NO` jeśli nie ma scdoca w systemie, ale wydaje się to trudno zrobić. Jeśli nie da się tego jakoś prosto zrobić, to może być tak jak jest.

Reply:
Fajny pomysł, spróbuję.